<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQMutexBased</class>
 <widget class="QMainWindow" name="TestQMutexBased">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>738</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestQMutexBased</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>线程</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnStartThread">
           <property name="text">
            <string>启动线程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDiceBegin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>开始掷骰子</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDicePause">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>暂停掷骰子</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStopThread">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>结束线程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>清空文本</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="LabPic">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/images/d0.jpg</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="LabA">
         <property name="text">
          <string>Thread状态：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
